pkg/network/usm/shared_libraries.go

```diff
@@ -207,14 +216,17 @@ func (w *soWatcher) Start() {
 		return
 	}
 
+	w.wg.Add(1)
 	go func() {
+		defer w.wg.Done()
 		defer cleanupExit()
-		defer w.processMonitor.Stop()
```

Comment:
why not calling stop anymore ?

Reply:
good catch, that will be removed in a future PR (the process monitor refactor)